README.md

```diff
@@ -743,6 +749,51 @@ Flags:
 -  `--platform=<PLATFORM>`              : convert content for a specific platform
 -  `--all-platforms`                    : convert content for all platforms (default: false)
 
+### :nerd_face: nerdctl image encrypt
+Encrypt image layers. See [`./docs/ocicrypt.md`](./docs/ocicrypt.md).
+
+Usage: `nerdctl image encrypt [OPTIONS] SOURCE_IMAGE[:TAG] TARGET_IMAGE[:TAG]`
+
+Example:
+```bash
+openssl genrsa -out mykey.pem
+openssl rsa -in mykey.pem -pubout -out mypubkey.pem
+nerdctl image encrypt --recipient=jwe:mypubkey.pem --platform=linux/amd64,linux/arm64 foo example.com/foo:encrypted
+nerdctl push example.com/foo:encrypted
+```
+
+CAUTION :warning:: This command only encrypt image layers, but does NOT encrypt [container config information such as `Env` and `Cmd`](https://github.com/opencontainers/image-spec/blob/v1.0.1/config.md#example).
+To see non-encrypted information, run `nerdctl image inspect --mode=native --platform=PLATFORM example.com/foo:encrypted` .
+
+Flags:
+-  `--recipient=<RECIPIENT>`      : Recipient of the image is the person who can decrypt (e.g., `jwe:mypubkey.pem`)
+-  `--dec-recipient=<RECIPIENT>`  : Recipient of the image; used only for PKCS7 and must be an x509 certificate
+-  `--key=<KEY>`                  : A secret key's filename and an optional password separated by colon
+-  `--gpg-homedir=<DIR>`          : The GPG homedir to use; by default gpg uses ~/.gnupg
+-  `--gpg-version=<VERSION>`      : The GPG version ("v1" or "v2"), default will make an educated guess
+-  `--platform=<PLATFORM>`        : Convert content for a specific platform
+-  `--all-platforms`              : Convert content for all platforms (default: false)
+
+
+### :nerd_face: nerdctl image decrypt
+Decrypt image layers. See [`./docs/ocicrypt.md`](./docs/ocicrypt.md).
+
+Usage: `nerdctl image decrypt [OPTIONS] SOURCE_IMAGE[:TAG] TARGET_IMAGE[:TAG]`
+
+Example:
+```bash
+nerdctl pull --unpack=false example.com/foo:encrypted
+nerdctl decrypt --key=mykey.pem example.com/foo:encrypted foo:decrypted
+```
+
+Flags:
+-  `--dec-recipient=<RECIPIENT>`  : Recipient of the image; used only for PKCS7 and must be an x509 certificate
+-  `--key=<KEY>`                  : A secret key's filename and an optional password separated by colon
```

Comment:
I suppose all these here are available at this point?
```
- <filename>:<password>
- <filename>:pass=<password>
- <filename>:fd=<file descriptor> (not available for rootless mode)
- <filename>:filename=<password file>
```

The above should probaby be

--key=<KEY>[:PWDDESC]

PWDDESC=<password>|pass:<password>|fd=<file descriptor>|filename

Reply:
fixed